tests/metrics/machine_learning/openvino.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'openvino benchmark'
+# https://github.com/v8/web-tooling-benchmark
+
+set -o pipefail
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+# TEST_NAME is required to collect results and name the workload container.
+TEST_NAME="openvino-bench"
+
+WORKLOAD="phoronix-test-suite batch-run openvino"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+
+IMAGE="docker.io/library/pts-openvino:latest"
+DOCKERFILE="${SCRIPT_PATH}/openvino-dockerfile/Dockerfile"
+
+TMP_DIR=$(mktemp --tmpdir -d openvino.XXXXXXXXXX)
+TEST_RESULTS_FNAME="${TMP_DIR}/openvino-results.json"
+
+readonly KATA_DIR="/opt/kata/share/defaults/kata-containers"
+readonly SRC_CONFIG="configuration.toml"
+
+# Variable used to store the initial configuration file name.
+# This file is again pointed to by kata once the script finishes.
+KATA_CONFIG_FNAME=""
+
+function restore_kata_config() {
+	rm -rf "${TMP_DIR}"
+
+	info "Restoring kata configuration"
+	[ -z "${KATA_CONFIG_FNAME}" ] && return
+
+	pushd "${KATA_DIR}"
+		sudo ln -sf "${KATA_CONFIG_FNAME}" "${SRC_CONFIG}"
+	popd
+}
+trap restore_kata_config EXIT
+
+clean_cache() {
```

Comment:
what about making this function as part of the common of metrics? this is being used in onednn and openvino ml benchmarks

Reply:
done, thank you @GabyCT